docs/getting-started/threat-intel-integrations.asciidoc

```diff
@@ -0,0 +1,50 @@
+[[es-threat-intel-integrations]]
+= Enable threat intelligence integrations
+
+The Threat Intelligence view gives you a streamlined way to collect threat intelligence data that you can use for threat detection and matching. You can connect to threat intelligence sources using an <<agent-ti-integration, {agent} integration>>, the <<ti-mod-integration, Threat Intel module>>, or a <<custom-ti-integration, custom integration>>.
+
+NOTE: The warning message about remaining threat intelligence sources is removed once you’ve enabled all available {elastic-sec} threat intelligence integrations.
```

Comment:
Unclear what warning message this refers to. I think it's talking about the message shown int the screenshot, but the phrasing throws me off: the message is not really a *warning,* and it doesn't refer to "remaining sources," just disabled ones.